paper/paper.md

```diff
@@ -0,0 +1,434 @@
+  - name: Michigan Neuroscience Institute, University of Michigan
+    index: 4
+  - name: Biophysics Graduate Program, University of Michigan
+    index: 5
+  - name: Department of Pathology, University of Michigan
+    index: 6
+  - name: School of Information, University of Michigan
+    index: 7
+  - name: Department of Environmental Health Sciences, University of Michigan
+    index: 8
+date: 30 Apr. 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+<!-- pasted from our PyCon 2020 talk proposal -->
+We are Bioinformatics PhD students at the University of Michigan who started a
+local chapter of Girls Who Code to provide a fun and supportive environment for
+high school women to learn the power of coding.
+Our goal was to cover basic coding topics and data science concepts through
+live-coding and hands-on practice.
+However, we couldn’t find a resource that exactly met our needs. Therefore, over the past
+three years, we have developed a curriculum and instructional format using
+Jupyter notebooks to effectively teach introductory Python for data science.
+This method, inspired by The Carpentries organization, uses bite-sized lessons
+followed by independent practice time to reinforce coding concepts, and
+culminates in a data science capstone project using real-world data.
+We believe our open curriculum is a valuable resource to the wider education
+community and hope that educators will use and improve our lessons, challenge
+problems, and teaching best-practices.
+Anyone can contribute to [our educational materials on GitHub](https://github.com/GWC-DCMB).
+
+
+# Statement of Need
+
+As women in the Department of Computational Medicine and Bioinformatics at the
+University of Michigan, we experience the gender gap first hand in our field.
+During the 1974-1975 academic year, women achieved 18.9% of total Bachelor's
+degrees in computer and information sciences in the US [@nces_digest_2012].
+By 1983-1984 this peaked at 37.1%, but fell to 17.6% by 2010-2011. We also see this
+national trend in the training of the next generation of Bioinformaticians at
+Michigan Medicine.
+Since accepting its first students in 2001, the U-M Bioinformatics Graduate
+Program has graduated 66 male and 22 female doctorates as of 2019.
+This disparity begins at the applicant level — during 2016-2019 the average
+percentage of females applying directly to the Bioinformatics PhD program is
+35.2%, and 41% for female applicants listing Bioinformatics as first or second
+choice in the biomedical PhD umbrella program PIBS.
+
+Previous research on women's educational experiences in STEM have produced
+various explanations for persistent gender disparities
+[@benbow_gender_2016].
+One explanation is that women in first year computer science courses rate
+themselves as being less experienced, prepared, or able to master the course
+material than the men [@fisher_undergraduate_1997].
+The majority of our organization's founding graduate students (all women) began
```

Comment:
Should we talk about how another explanation could be due to the effects of negative stereotypes (e.g.  girls are not as good as boys in math, and scientific work is better suited to boys and men)? https://www.aauw.org/app/uploads/2020/03/why-so-few-research.pdf (pg 54-59). We want to expose young women to successful female role models to help counter such stereotypes and help them see that they can be successful too.

Reply:
Help wanted! Anyone want to try to address this?

Reply:
Done